src/Sulu/Bundle/ResourceBundle/Resources/public/js/components/condition-selection/main.js

```diff
@@ -449,12 +455,59 @@ define([], function() {
                 case 'simple':
                     return createSimpleInput.call(this, value, constants.valueInputClass);
                 default:
-                    this.sandbox.logger.error('Input type "' + type + '" is not supported!');
-                    break;
+                    return createGenericInput.call(this, value, constants.valueInputClass, operator, parameters);
             }
         },
 
         /**
+         * Decides which input should be displayed for the given condition
+         */
+        getInputValue = function(operator, $row) {
+            switch (operator.inputType || '') {
+                case 'date':
+                case 'datepicker':
+                    return this.sandbox.dom.val(this.sandbox.dom.find('.' + constants.valueInputClass + ' input', $row));
+                case '':
+                case 'select':
+                case 'boolean':
+                case 'radio':
+                case 'checkbox':
+                case 'simple':
+                    return this.sandbox.dom.val(this.sandbox.dom.find('.' + constants.valueInputClass, $row));
+                default:
+                    return $('.' + constants.valueInputClass, $row).data('value');
+            }
+        },
+
+        /**
+         * Start a component for given operator.
+         *
+         * @param value
+         * @param cssClass
+         * @param operator
+         * @param parameters
+         *
+         * @returns {*|jQuery|HTMLElement}
+         */
+        createGenericInput = function(value, cssClass, operator, parameters) {
+            var $el = $('<div class="' + cssClass + '"/>');
+
+            this.sandbox.start([
+                {
+                    name: 'condition-selection/' + operator.inputType + '@suluresource',
```

Comment:
yeah but there we use directly require not aura ... in aura it is not possible todo it in that way!